setup.py

```diff
@@ -1,9 +1,26 @@
 #!/usr/bin/python2
 
-from distutils.core import setup
 import os
 import sys
 
+from distutils.core import setup
+from distutils.command.install_data import install_data as _install_data
+from distutils.file_util import move_file
+
+class install_data(_install_data):
+    def run(self):
+        _install_data.run(self)
+
+        for old in self.get_outputs():
+            if old.find('/usr/') > -1 and old.endswith('.py'):
+                new = os.path.splitext(old)[0]
+                # move_file doesn't have a way to tell it to overwrite, so remove
+                # the existing copy to accomplish the same thing.
+                if os.path.isfile(new) and os.path.exists(new):
+                    os.unlink(new)
+                move_file(old, new)
```

Comment:
@bcl this piece of code is supposed to remove the `.py` suffix when installing. I did limited testing but it looks like it works.